benchmarks/dynamo/common.py

```diff
@@ -3478,6 +3477,30 @@ def run(runner, args, original_dir=None):
             # Go back to main branch
             repo.git.checkout(main_branch)
     elif args.only:
+        def write_csv_when_exception(name, status):
+            placeholder_batch_size = 0
+            if args.accuracy:
+                headers = ["dev", "name", "batch_size", "accuracy"]
+                rows = [
+                    [device, name, placeholder_batch_size, status]
+                    for device in args.devices
+                ]
+            elif args.performance:
+                headers = ["dev", "name", "batch_size", "speedup", "abs_latency"]
+                rows = [
+                    [device, name, placeholder_batch_size, 0.0, 0.0]
+                    for device in args.devices
+                ]
+            else:
+                headers = []
+                rows = [
+                    [device, name, placeholder_batch_size, 0.0]
+                    for device in args.devices
+                ]
+
+            for row in rows:
+                output_csv(output_filename, headers, row)
```

Comment:
Nit (avoid code repeptiion)
```suggestion
                generator = lambda device:  [device, name, placeholder_batch_size, status]
            elif args.performance:
                headers = ["dev", "name", "batch_size", "speedup", "abs_latency"]
                generator = lambda device: [device, name, placeholder_batch_size, 0.0, 0.0]
            else:
                headers = []
                generator = lambda device: [device, name, placeholder_batch_size, 0.0]

            for row in [generator(device) for device in args.devices]:
                output_csv(output_filename, headers, row)
```

Reply:
```
  Warning (FLAKE8) E731
    do not assign a lambda expression, use a def
    See https://www.flake8rules.com/rules/E731.html
 ```
 linter doesn't like it.

Reply:
Well, just define a generator function then :)